src/theory/strings/core_solver.cpp

```diff
@@ -981,7 +981,7 @@ void CoreSolver::processNEqc(std::vector<NormalForm>& normal_forms)
   bool set_use_index = false;
   Trace("strings-solve") << "Possible inferences (" << pinfer.size()
                          << ") : " << std::endl;
-  unsigned min_id = 9;
+  Inference min_id = Inference::NONE;
```

Comment:
We should perhaps move NONE to the end of the enum just to make sure its clear that the heuristic on order inferences is consistent.  Right now, its weird setting min_id to NONE=0 initially.

Reply:
I have moved it to the end (note that this does not change the semantics of the code because we have `set_use_index` to detect whether or not the values have been set).